port_ocean/core/defaults/initialize.py

```diff
@@ -205,12 +246,9 @@ async def _initialize_defaults(
         raise ExceptionGroup(str(e), e.errors)
 
 
-def initialize_defaults(
+def initialize_integration(
     config_class: Type[PortAppConfig], integration_config: IntegrationConfiguration
 ) -> None:
-    try:
-        asyncio.new_event_loop().run_until_complete(
-            _initialize_defaults(config_class, integration_config)
-        )
-    except Exception as e:
-        logger.debug(f"Failed to initialize defaults, skipping... Error: {e}")
+    asyncio.new_event_loop().run_until_complete(
+        _initialize_integration(config_class, integration_config)
+    )
```

Comment:
Why did u femove exception handling?

Reply:
don't we want it to fail in case the mapping couldn't be applied? i think it will confuse the user if we'll continue normally and resyncs wouldn't work